patches/tModLoader/Terraria/Localization/Content/en-US/tModLoader.json

```diff
@@ -438,24 +438,25 @@
 		"OneDriveHint": "\n\nTip: Try installing/enabling OneDrive. Right click your Documents folder and enable \"Always save on this device\"",
 		"AntivirusHint": "\n\nTip: If you have an antivirus, try adding Terraria to its whitelist/exclusion list",
 		"TypeInitializationHint": "\n\nTip: Restart the game. If this issue persists, try uninstalling Terraria completely, then reinstalling through Steam, and then reinstalling tModLoader",
+		// Tag descriptions by Webmilio, Tomat, Solxan, and Setnour6.
 		"TagsContent": "New Content",
-		"TagsContentDescription": "This is a placeholder description. Don't Translate!",
+		"TagsContentDescription": "Adds new content (i.e. items, weapons, npcs, etc.) to the game",
 		"TagsUtility": "Utilities",
-		"TagsUtilityDescription": "This is a placeholder description. Don't Translate!",
+		"TagsUtilityDescription": "Adds utility-oriented content or features",
 		"TagsLibrary": "Library",
-		"TagsLibraryDescription": "This is a placeholder description. Don't Translate!",
+		"TagsLibraryDescription": "Provides additional API & features useful to modders",
 		"TagsQoL": "Quality Of Life",
-		"TagsQoLDescription": "This is a placeholder description. Don't Translate!",
+		"TagsQoLDescription": "Features or changes that reduce time spent on typically less fun tasks.",
 		"TagsGameplay": "Gameplay Tweaks",
-		"TagsGameplayDescription": "This is a placeholder description. Don't Translate!",
+		"TagsGameplayDescription": "Tweaks that change or improve vanilla gameplay",
 		"TagsAudio": "Audio Tweaks",
-		"TagsAudioDescription": "This is a placeholder description. Don't Translate!",
+		"TagsAudioDescription": Tweaks that change existing audio (i.e. new boss themes, etc.)",
```

Comment:
Missing a " at the start of the string.

Reply:
How did that happen? Oh well, I fixed it.